array/main.js

```diff
@@ -0,0 +1,107 @@
+const log = console.log;
+let sports = ["cricket", "football", "pool", "kabaddi"]; //declare array
+log(sports[0]); // print cricket
+log(sports[4]); // undefined
+
+
+let last = sports[sports.length - 1];
+log(last); // O/P kabaddi
+
+// looping array
+sports.forEach(function (item, index, array) {
+  log(item, index); //cricket 0 football 1 pool 2 kabaddi 3
+});
+
+// add to the end of an array
+let addSport = sports.push("kho-kho");
+log(sports);
+
+// remove form the end of an array
+let lastRemove = sports.pop(); // remove kho-kho
+log(sports);
+
+// remove form the front of an array
+let firstRemove = sports.shift(); //remove cricket
+log(sports);
+
+// add to the front of an array
+let addFront = sports.unshift("cricket");
+log(sports);
+
+// find index of an item in the array
+let position = sports.indexOf("pool");

```

Comment:
No need of two lines here.
Directly do `log(sports.indexOf('pool'));`.
Similarly do it for some statements below.